src/components/Form.js

```diff
@@ -1,13 +1,30 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/book';
 
 function Form() {
+  const dispatch = useDispatch();
+  const [author, setAuthor] = useState('');
+  const [title, setTitle] = useState('');
+  const [categories, setCategories] = useState('');
+  // const categories = [Horror, Biopic, Documentry, Adventure];
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    dispatch(addBook({
+      author, title, categories, id: Date.now(),
+    }));
```

Comment:
- [ ] **[OPTIONAL]** For optimal User experience, it would be nice to reset the form after submission by setting your states to empty string like so: `setTitle('')` and so on.